docs/how-to-use-with-rails-backend.md

```diff
@@ -85,7 +85,7 @@ Create a `package.json` at the root of the Rails project with these contents:
     "npm-run-all": "^4.1.3"
   },
   "engines": {
-    "node": "10.15.0",
+    "node": "^14.16.0",
```

Comment:
Rather than lock to a specific node version, allow anything that is semver-compatible.